sdks/python/apache_beam/runners/common.pxd

```diff
@@ -87,15 +87,15 @@ cdef class DoFnRunner(Receiver):
 
 
 cdef class OutputProcessor(object):
-  @cython.locals(windowed_value=WindowedValue)
+  @cython.locals(windowed_value=WindowedValue, output_element_count=cython.int)
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Please use `int64_t` after importing `from libc.stdint cimport int32_t, int64_t` at the top of the file.